includes/class-wc-payments-express-checkout-button-display-handler.php

```diff
@@ -57,10 +57,11 @@ public function __construct( WC_Payment_Gateway_WCPay $gateway, WC_Payments_Paym
 			add_action( 'woocommerce_after_add_to_cart_form', [ $this, 'display_express_checkout_buttons' ], 1 );
 			add_action( 'woocommerce_proceed_to_checkout', [ $this, 'display_express_checkout_buttons' ], 21 );
 			add_action( 'woocommerce_checkout_before_customer_details', [ $this, 'display_express_checkout_buttons' ], 1 );
+			add_action( 'before_woocommerce_pay_form', [ $this, 'add_pay_for_order_params_to_js_config' ] );
 
 			if ( $is_payment_request_enabled ) {
```

Comment:
`is_payment_request_enabled` is only true for GPay and ApplePay flow. If the merchant only has WooPay EPB enabled, the button will not show up on the pay for order page. 
We can remove this condition if both flows now support the pay for order flow.

Reply:
>both flows now support the pay for order flow.

Do you mean `$is_payment_request_enabled` is for ApplePay and Gpay?  If so, I think we should change this condition to `$is_woopay_enabled`?

Reply:
I see what you mean! [5cc1e39](https://github.com/Automattic/woocommerce-payments/pull/5903/commits/5cc1e3988a33d3c4888ace22114ffa7ad2262c34)